tests/specs/utils/http.spec.js

```diff
@@ -114,18 +114,22 @@ describe('HTTP', () => {
             });
         });
 
+        function randomIntBetween(min, max) {
+            return Math.floor(Math.random() * (max - min)) + min;
+        }
+
         describe('#handleStatus', () => {
-            for (let status = 200; status < 300; status++) {
+            [200, randomIntBetween(201, 299), 299].forEach((status) => {
```

Comment:
which is basically the case because 202 and 204 are special cases for the body handling part.